options.go

```diff
@@ -1211,3 +1375,25 @@ func (opts *Options) Destroy() {
 	opts.env = nil
 	opts.bbto = nil
 }
+
+// GetTickerCount
+func (opts *Options) GetTickerCount(ticker Ticker) uint64 {
+	retval, _ := C.gorocksdb_get_ticker_count(opts.c, C.uint(ticker))
+
+	return uint64(retval)
+}
+
+func (opts *Options) GetAndResetTickerCount(ticker Ticker) uint64 {
+	retval, _ := C.gorocksdb_get_and_reset_ticker_count(opts.c, C.uint(ticker))
+
+	return uint64(retval)
+}
+
+func (opts *Options) SetStatsLevel(sl StatsLevel) {
```

Comment:
Do we ever need to set stats?

Reply:
We should never need to, but that's not what SetStatsLevel does.  I've documented SetStatsLevel with the appropriate information.